spec/features/gis_accessioning_spec.rb

```diff
@@ -83,15 +87,53 @@
     # look for expected files produced by GIS workflows
     files = all('tr.file')
     expect(files.size).to eq 9
-    expect(files[0].text).to match(%r{AirMonitoringStations.shp application/vnd.shp 8.14 KB})
-    expect(files[1].text).to match(%r{AirMonitoringStations.shx application/vnd.shx 2.39 KB})
-    expect(files[2].text).to match(%r{AirMonitoringStations.dbf application/vnd.dbf 40.8 KB})
-    expect(files[3].text).to match(%r{AirMonitoringStations.prj text/plain 468 Bytes})
-    expect(files[4].text).to match(%r{preview.jpg image/jpeg 2\d.\d KB})
-    expect(files[5].text).to match(%r{AirMonitoringStations.shp.xml application/xml 6\d.\d KB})
-    expect(files[6].text).to match(%r{AirMonitoringStations-iso19139.xml application/xml 2\d.\d KB})
-    expect(files[7].text).to match(%r{AirMonitoringStations-iso19110.xml application/xml 1\d.\d KB})
-    expect(files[8].text).to match(%r{AirMonitoringStations-fgdc.xml application/xml 5.\d+ KB})
+
```

Comment:
This was an interesting situation. Playwright did not match the text across table cells the way this was originally written. For instance I got:
```
- expected: 'AirMonitoringStations.shp\t\t\tapplication/vnd.shp 8.14 KB'
- got: 'AirMonitoringStations.shp\t\t\tapplication/vnd.shp 8.14 KB'
```
with some variation on the whitespace between the tabs.

By instead verifying the cells and then the individual text, this seems a bit clearer and performant (but that could just be my perception)